tests/waku_node/test_wakunode_relay_rln.nim

```diff
@@ -147,6 +147,12 @@ suite "Waku RlnRelay - End to End":
           "WakuRelay protocol is not mounted, cannot mount WakuRlnRelay"
 
     # FIXME: fails on macos
+    #[
+      Unhandled defect: 
+      /Users/runner/work/nwaku/nwaku/tests/waku_node/test_wakunode_relay_rln.nim(59, 18) 
```

Comment:
can probably add a sleep. this is weird since the pr which merged these changes in did not suffer from this. nonceLimit shouldn't be 0

Reply:
Testing sleep hypothesis.
In any case, there are three different errors, there might be different origins to them.

Reply:
I've added some awaits here and there, not in all failing tests, yet this last run hasn't triggered test failures. There's still an exception:
```
2024-03-20T15:29:25.3026940Z [Summary] 562 tests run (1018.6s): 554 OK, 0 FAILED, 8 SKIPPED
2024-03-20T15:29:25.8017230Z stack trace: (most recent call last)
2024-03-20T15:29:25.8046630Z /Users/runner/work/nwaku/nwaku/vendor/nimbus-build-system/vendor/Nim/lib/system/nimscript.nim(429, 18)
2024-03-20T15:29:25.8050330Z /Users/runner/work/nwaku/nwaku/waku.nimble(82, 8) testTask
2024-03-20T15:29:25.8051690Z /Users/runner/work/nwaku/nwaku/waku.nimble(54, 8) test
2024-03-20T15:29:25.8053620Z /Users/runner/work/nwaku/nwaku/vendor/nimbus-build-system/vendor/Nim/lib/system/nimscript.nim(273, 7) exec
2024-03-20T15:29:25.8057400Z /Users/runner/work/nwaku/nwaku/vendor/nimbus-build-system/vendor/Nim/lib/system/nimscript.nim(273, 7) Error: unhandled exception: FAILED: build/all_tests_waku [OSError]
2024-03-20T15:29:25.8726590Z make: *** [testwaku] Error 1
2024-03-20T15:29:25.9231280Z ##[error]Process completed with exit code 2.
```
But I can't figure out what it might be related to. Any ideas?

Reply:
you have to initialize the node properly differently for rln-v1 and v2, please see the test_wakunode_rln_relay.nim test